api/context.go

```diff
@@ -102,7 +106,9 @@ func (h handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	c.RequestId = model.NewId()
 	c.IpAddress = GetIpAddress(r)
 	c.TeamId = mux.Vars(r)["team_id"]
-	h.isApi = IsApiCall(r)
+
+	// this is being set by the handler functions, no need to re-assign here
+	//h.isApi = IsApiCall(r)
```

Comment:
As far as i can tell every route as one of the following:

* ApiAppHandler
* AppHandler
* AppHandlerIndependent
* ApiUserRequired
* ApiUserRequiredActivity
* UserRequired
* ApiAdminSystemRequired
* ApiAppHandlerTrustRequester
* ApiUserRequiredTrustRequester
* ApiAppHandlerTrustRequesterIndependent

and each one of them assign the isApi attribute for the handler